ZFSin/zfs/module/zfs/zfs_vnops_windows_mount.c

```diff
@@ -750,8 +750,8 @@ VOID InitVpb(__in PVPB Vpb, __in PDEVICE_OBJECT VolumeDevice)
 										+0xffff928400000000
 
 */
```

Comment:
add a comment to justify the change, like: the LPCWSTR could be non-NULL terminated when passed and causes the buffer to be overran.

Reply:
I will change UNICODE_STRING to PCUNICODE_STRING, and will add your justification propose, why the  LPCWSTR was changed.